apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -702,12 +766,12 @@ def grade(element_html, data):
             -1 if num_initial_correct == len(submission) else num_initial_correct
         )
 
-        if partial_credit_type == "none":
+        if partial_credit_type is PartialCreditType.NONE:
             if num_initial_correct == true_answer_length:
```

Comment:
```suggestion
            final_score = 1 if num_initial_correct == true_answer_length else 0
```

Reply:
I don't want to include that refactor here, I'm trying to only do typing stuff here.